### ISSUE TITLE
When using inherited users sign up and registration links displayed

### ISSUE THREAD

**Issue by davidnewcomb** [bizley/yii2-podium#125]
When the user is logged out the forum displays links for "Sign in to reply" and "Register new account". Clicking on either of these links loads a new page with a flash to say "Please use application's Register form to sign up".

If you know that they must use the other sign up page then there is no need to display the links. Or at least there should be some way to switch them off or configure them to point to the application's login and registration pages. (I'm voting for the switch!)

**Comment by bizley** [comment]
Right. I've removed the links from the top bar but left the button links.

**Comment by davidnewcomb** [comment]
When you say "top bar", do you mean the top navigational bar? In my application I've overloaded your navbar to show my navigation bar where your nav is a sub-part of my nav.
The button links are "in page" an so I'd have to overload the whole page (and all the other pages) in order to remove them.

**Comment by bizley** [comment]
Exactly. I need to fix this.

**Comment by fedornabilkin** [comment]
Т.к. регистрация и авторизация являются неотъемлемой частью приложения и форума, то эти ссылки можно добавить в конфигурацию и перенаправлять пользователя по ним. Если такие ссылки не указаны в конфигурации, перенаправлять пользователя на главную страницу приложения и добавлять flash сообщение. Как правило, на главной странице обычно всегда есть кнопки регистрации и авторизации.

**Comment by davidnewcomb** [comment]
@fedornabilkin wrote:

Because registration and authorization are an integral part of the application and offline, these links can be added to the configuration and redirect the user on them. If such links are not specified in the configuration, redirect the user to the main page of the application and add the flash message. As a rule, usually there is always a registration button and login on the home page.
